.github/workflows/release.yaml

```diff
@@ -0,0 +1,47 @@
+name: Build and push a release image
+on:
+  push:
+    tags:
+      - '**'
```

Comment:
So you plan to trigger build + push by pushing a tag? I would have expected to e.g. pass-in version via input + have the pipeline publish a tag (there might be cases where build will fail, which will require resetting tags, if I understand the approach correctly - which does not seem like a good practice to me) :-/

Reply:
Sounds good. I've re-done it using workflow_dispatch inputs that should be provided, and new release will be done by the GH action itself.